composer.json

```
@@ -26,5 +26,24 @@
 			"modules/",
 			"settings/"
 		]
+	},
+	"scripts": {
+		"test":"vendor/bin/phpunit",
```

What do you think of this?

```suggestion
		"test":"vendor/bin/phpunit --colors=always",
```